### PR TITLE
Remove check letter from SDS identifier

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Removed the check letter from the identifier as ru_ref in SDS shouldn't include them.

### How to review
Use this branch with the [Runner branch](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1280) to ensure launch of SDS still works.

### CODE REVIEW

**Review by petechd — APPROVED** [review]

File: app/test_app.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def test_get_sds_data_found():
         "/v1/unit_data",
         params={
             "dataset_id": "c067f6de-6d64-42b1-8b02-431a3486c178",
-            "identifier": "1234678912A",
+            "identifier": "1234678912",
```
Comment (katie-gardner) [inline app/test_app.py]: Should this be 12346789012? same below

Comment (katie-gardner) [reply app/test_app.py]: The test will still pass since its unused but for clarity?

Comment (MebinAbraham) [reply app/test_app.py]: Updated for consistency. We should consider updating the mock to check that value, at the moment the dataset is hardcoded but it doesn't need to be. Makes adding new schemas easier also.

**Review by katie-gardner — APPROVED** [review]
Looks good 👍 just needs same change in identifier when main is merged in with the 3rd dataset

**Review by berroar — APPROVED** [review]

**Pull request merged**